linux_os/guide/system/accounts/accounts-session/accounts_tmout/bash/sle12.sh

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_sle
```

Comment:
I think these `accounts_tmout` can be removed as per: https://github.com/ComplianceAsCode/content/pull/6631#discussion_r584669502

Reply:
Let's leave these changes as per https://github.com/ComplianceAsCode/content/pull/6634/files#r587065689